lib/optimize/SplitChunksPlugin.js

```diff
@@ -313,18 +314,14 @@ module.exports = class SplitChunksPlugin {
 		return false;
 	}
 
-	/**
-	 * @param {Compiler} compiler webpack compiler
```

Comment:
I had to remove this until `tapable` defs are fixed

Reply:
We need to fix that before this is merged